src/server/google-cloud.md

```diff
@@ -50,6 +50,15 @@ so we recommend that you **use Cloud Run** instead of App Engine for new
 server-side Dart code.
 If you _want_ to use App Engine, consider using the [`appengine` package][].
 
+## Functions Framework for Dart
+
+The Dart Functions Framework is a Google-sponsored open source 
+FaaS (Function as a Service) project that makes it easy to write Dart functions
+instead of server applications for handling web requests.
+Using the framework, you can create functions that will handle HTTP requests
+and [CloudEvents] and deploy them to Google Cloud.
```

Comment:
```suggestion
and [CloudEvents][] and deploy them to Google Cloud.
```

Reply:
👍

Reply:
...I wish this was enforced w/ CI if it's the convention 🤷  (good catch, @parlough – I'm ranting at the sky 😁 )

Reply:
I've been looking into enforcing some of our Markdown and Jekyll styles with CI, but I don't have anything yet :)